src/Azure.Functions.Cli/Actions/AuthActions/BaseAuthAction.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Linq;
+using Azure.Functions.Cli.Actions.AzureActions;
+using Fclp;
+
+namespace Azure.Functions.Cli.Actions.AuthActions
+{
+    abstract class BaseAuthAction : BaseAzureAction
```

Comment:
You don't really need this class since it's empty, right? you can just drive from `BaseAzureAction` if you need the access token, or were you planning on adding some common code here?